README.md

```diff
@@ -118,6 +118,24 @@ helm install capi-operator capi-operator/cluster-api-operator --create-namespace
 ```
 This will take some time to install and can be verified it is complete by seeing two ready pods in the `azure-infrastructure-system` namespace. 
 
+In order to install CAPI Operator with additional CRDs, you will need to use `helm install` with a `values.yaml` file and customize the `azureserviceoperator-controller-manager` deployment like so:
```

Comment:
```suggestion
In order to install CAPI Operator with additional CRDs, `helm install` must use a `values.yaml` file since the commands cannot be passed on the command line.  documentdb and managedidentity CRDS are added by default in the provided [values.yaml file](./gitops/environments/default/addons/cluster-api-provider-azure/values.yaml) and can be optionally customized if desired.  For more information read the first section of [CAPZ versus Crossplane](./docs/capz-or-crossplane.md#cluster-api-provider-for-azure-capz-and-azure-service-operator-aso).
```